### PR TITLE
multi: remove dead code

### PR DESCRIPTION
**Pull request by joostjager**

### CODE REVIEW

File: contractcourt/briefcase.go [hunk 1]
```diff
@@ -176,11 +176,6 @@ func (a ArbitratorState) String() string {
 	}
 }
 
-// resolverType is an enum that enumerates the various types of resolvers. When
-// writing resolvers to disk, we prepend this to the raw bytes stored. This
-// allows us to properly decode the resolver into the proper type.
-type resolverType uint8
```
Comment (wpaulino) [inline contractcourt/briefcase.go]: I think the intent of this was to mark the constants below with this type.

File: lnwallet/channel.go [hunk 2]
```diff
@@ -109,7 +109,7 @@ type channelState uint8
 const (
 	// channelPending indicates this channel is still going through the
 	// funding workflow, and isn't yet open.
-	channelPending channelState = iota
+	channelPending channelState = iota // nolint: unused
```
Comment (wpaulino) [inline lnwallet/channel.go]: I don't think this enum is actually serving a purpose at all, perhaps we can remove it completely?

Comment (joostjager) [reply lnwallet/channel.go]: What do you mean exactly? `channelState` is used. This particular value not, but I didn't want to remove it to change the numbering (serialization in mind perhaps)

File: contractcourt/briefcase.go (hunk 1 above)
Comment (joostjager) [reply contractcourt/briefcase.go]: Ok, left in place and added to the values

File: lnwallet/channel.go (hunk 2 above)
Comment (wpaulino) [reply lnwallet/channel.go]: I don't think `LightningChannel` is serialized to disk. I was referring more to if you look at the uses of this enum (not just `channelPending`) within `LightningChannel`, it doesn't seem to be doing much.

Comment (joostjager) [reply lnwallet/channel.go]: I gave it a shot, but the closed state is still set and checked. I'd say, "not in this pr" ?

Comment (wpaulino) [reply lnwallet/channel.go]: Sure can be left for a follow-up. I think it's just outdated code.

**Pull request merged**